src/components/pod/pod.test.js

```diff
@@ -65,13 +68,15 @@ const SoftDeletePodWithChildren = () => (
 context("Testing Pod component", () => {
   describe("should render Pod component", () => {
     it.each([
-      [true, "1px solid rgb(204, 214, 219)"],
-      [false, "0px none rgb(0, 0, 0)"],
+      [true, 1, "solid", "rgb(204, 214, 219)"],
+      [false, 0, "none", "rgba(0, 0, 0, 0.9)"],
     ])(
       "should check when border is %s the border value is %s for Pod component ",
-      (boolVal, border) => {
+      (boolVal, px, style, color) => {
         CypressMountWithProviders(<PodComponent border={boolVal} />);
-        podBlock().should("have.css", "border", border);
+        podBlock().then((elem) => {
+          checkOutlineCss(elem, 1, "border", style, color);
```

Comment:
Should this be:
          checkOutlineCss(elem, px, "border", style, color);
?